susemanager/src/mgr_sync/config.py

```diff
@@ -65,11 +65,17 @@ def _parse_config(self):
 
         # Read /etc/rhn/rhn.conf if any
         if os.access(Config.RHNFILE, os.R_OK):
-            self._config.merge(ConfigObj(Config.RHNFILE))
+            try:
+                self._config.merge(ConfigObj(Config.RHNFILE))
+            except SyntaxError as ex:
+                raise SyntaxError("Error while parsing file{0}: {1}".format(Config.RHNFILE, ex)) from ex
```

Comment:
Are we missing a whitespace here to produce a nice message? Maybe I'm missing something, but should be like:

```suggestion
                raise SyntaxError("Error while parsing file {0}: {1}".format(Config.RHNFILE, ex)) from ex
```

Reply:
Done. Thank you!